travis/shared_configs/python-tester-conda-latest.yml

```diff
@@ -38,7 +38,6 @@ jobs:
         - micromamba list
       script:
         - |
-          set -e
```

Comment:
Did we confirm that removing this still reports failures as expected?  If so, what did this do in the first place?..

Reply:
Yeah, I was getting normal failures on typhos- maybe it does something on other repos? Perhaps with the concurrent tests thing?

Reply:
I think in the cases where there's more than one command it just would fail the job early and not combine coverage

Seems an unnecessary line overall